src/codegen/code_context.cpp

```diff
@@ -19,7 +19,7 @@
 #include "llvm/Support/raw_os_ostream.h"
 #include "llvm/Support/TargetSelect.h"
 #include "llvm/Transforms/Scalar.h"
-#if LLVM_VERSION_GE(3, 9)
+#if LLVM_VERSION_GE(3, 9) || LLVM_VERSION_GE(4, 0)
```

Comment:
I don't think you need this change since the macro checks for LLVM versions >= 3.9 already.